supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -429,6 +455,21 @@ Always spell out the full product name of the host, and do not capitalize the te
 
 *See also*: xref:red-hat-network-satellite-server[Red Hat Network Satellite Server]
 
+[discrete]
+[[red-hat-openjdk]]
+==== image:images/no.png[no] Red Hat OpenJDK (noun)
+*Description*: Do not use _Red Hat OpenJDK_ to refer to the Red Hat distribution of the Open Java Development Kit (OpenJDK).
+
+OpenJDK is a registered trademark of Oracle and/or its affiliates.
```

Comment:
See above about "and/or"

Reply:
Thanks so much for reviewing Miriam :hugs: 

I was tempted to do same but... 

I copied that specific sentence verbatim from the [Red Hat legal notices](https://access.redhat.com/documentation/en-us/red_hat_software_certification/8.64/html/red_hat_software_certification_quick_start_guide/legal-notice) .

Because it's a legally approved statement that appears in many other online publications, including an IBM-style guideline about Java, so I deliberately chose not to remove the `/or` even though it bugged me. 
WDYT?

Reply:
@mportman12 - After discussion with @joaedwar, I adjusted the statement as this guideline with the rationale that this is for internal use only and it's much better to practice what we preach. 
Thanks so much all for your input on this one :green_heart: